students/PeterB/Session2/fizzbuzz.py

```diff
@@ -0,0 +1,11 @@
+for fizzbuzz in range(100):
+    if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
+        print("fizzbuzz")
+        continue
+    elif fizzbuzz % 3 == 0:
+        print("fizz")
+        continue
+    elif fizzbuzz % 5 == 0:
+        print("buzz")
+        continue
+    print(fizzbuzz)
```

Comment:
We only want to print out the number if it doesn't fit the categories above. In this case, it is always printing.

Note: You don't want the continues in the if statements. Once we hit one of the "ifs", we don't want to do anything else for that number